terraform/environments/production/main.tf

```diff
@@ -510,15 +510,18 @@ locals {
     # Emails
     {
       name  = "OUTBOUND_EMAIL_ADAPTER"
-      value = "Elixir.Swoosh.Adapters.Postmark"
+      value = "Elixir.Swoosh.Adapters.Mailgun"
     },
     {
       name  = "OUTBOUND_EMAIL_FROM"
-      value = "support@firezone.dev"
+      value = "notifications@firezone.dev"
```

Comment:
We should not use email addresses that users can't reply to or that we will never keep an eye on: https://postmarkapp.com/blog/no-no-reply.

Reply:
Or we could set the `reply-to` to `support@<domain>` if we want

Reply:
I don't remember if we have `notifications@` configured, @jamilbk what is the email users should reply to?

Reply:
`notifications@` alias is not set up. For could you send from that address but then have the reply-to set to `support@firezone.dev`?

Reply:
That will allow users to filter based on sender but then also reply to create a support ticket if needed

Reply:
Yeah, I believe there is a provider option that can be set.  I'll check and update the PR with that or if it's not possible I'll just rollback this specific change until we setup the `notification@` alias.

Reply:
On second thought, I just created the `notifications@` alias since we might want to differentiate internally between support emails and replies to notifications (both route to the same place)